tests/json_requests/json_03.json

```diff
@@ -20,21 +20,21 @@
         "requesterSignature": "",
         "inData": [
 		   {"index": 2,
-                    "dataHash": "12ab34cd",
+                    "dataHash": "",
```

Comment:
workerEncryptionKey is also hardcoded in json file. This could also be removed. Either do in in same PR and update the commit message or raise a new PR

Reply:
Ok.